### ISSUE TITLE
Make a heritage command

### ISSUE THREAD

**Issue by Cyborger** [Cyborger/seconde-5-bot#29]
**This command would consist on fixing a really big problem for servers with a single owner**, let me explain myself : 

If as a single server owner, **if I loose access to my Discord account or if I'm not capable of managing the Discord server anymore** it's gonna be chaos on the server.

To prevent that, the first idea of the command is to fix that by allowing certain users, after a certain amount of time and if the owner has not rejected the request, to be granted a high rank on the server.

**In other words and with an example :** 
I'm a trustfully user on the server and someone who knows the owner : 
1. I type the command
2. The bot says to me if it is possible for me to run the command, if so, the bot tells me the time I need to wait to prevent bad usage of this command.
3. I wait and... Two options : 
    - The bots grants me the rank if the owner didn't reacted to this request.
    - The owner has connected and rejected my request and I don't get the rank.
